Gemfile

```diff
@@ -3,12 +3,15 @@ source "http://rubygems.org"
 gemspec
 
 # Uncomment and change rails version for testing purposes
-# gem "rails", "~> 5.2.0.beta2"
+# gem "rails", "~> 5.0.0"
```

Comment:
Might be a good use case for the [appraisals gem](https://github.com/thoughtbot/appraisal)

Reply:
We used to use the appraisals gem. Not too long before I became the maintainer, we stopped. I don't remember the reasoning for dropping appraisal. 

Another option that might help us is the https://github.com/clio/ten_years_rails gem, which helps you run the current and next future Rails version.